examples/incremental-learning-dog-croissants-classification.Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mindspore/mindspore-cpu:1.8.1
```

Comment:
There are two copies of Dockerfile, I think there should only one Dockerfile.
Please keep one, and remove another one.

Reply:
done